homework6.4/homework6.4.py

```diff
@@ -0,0 +1,5 @@
+import numbers
```

Comment:
Redundant import . 
There is no Task description 
Have you checked code with linter ?